src/main/resources/assets/styles/_publicationArchive.scss

```diff
@@ -10,6 +10,16 @@
     .publication-archive-body {
       & > * {
         max-width: 580px;
-      }  
+      }
+    }
+    .defineContentPhrase{
+      font-family: Roboto,sans-serif!important;
+      font-stretch: normal;
+      font-weight: 400;
+      font-weight: 700;
+      margin: 0 0 0 0;
+      color: #162327;
```

Comment:
No need for this line. The base color is already correct!

Reply:
removed color: #162327;